yang-lsp/io.typefox.yang/src/main/java/io/typefox/yang/utils/YangPathProvider.xtend

```diff
@@ -57,15 +66,11 @@ class YangPathProvider {
 		return resourceDescriptions
 	}
 
-	private def void process(File file, ResourceSet resourceSet) {
+	private def void process(File file, ResourceSet resourceSet, List<File> ignoredFiles) {
 		try {
-			if (!file.canRead)
+			if (!file.canRead) {
 				return
-			else if (file.isDirectory)
-				file.listFiles.forEach[process(resourceSet)]
-			else if (file.name.endsWith('.yang'))
-				resourceSet.getResource(URI.createFileURI(file.absolutePath), true)
-			else if (file.name.endsWith('.zip')) {
+			} else if (file.name.endsWith('.zip')) {
```

Comment:
I think you need to check if it is a file `isFile`.  Otherwise it will match directories named e.g. `some.zip`

Reply:
Fixed in the latest patch